Syndie Lavaland Base Flavortext Edit

ExcessiveUseOfCobblestone opened the pull request:
Why: Because people were outing antags [not traitors] via this and so I want to make it clear that's not really ok by explicitly adding that they should assist baddies instead of harm them [even those that aren't syndicate-aligned].

:cl: Cobby
add: Syndicate Lavabase now has more explicit instructions when it comes to outing non-syndicate affiliated enemies of Nanotrasen.
/:cl:


"Remember, an enemy of our enemy is a friend, so also provide support to those that hinder Nanotrasen unless commanded otherwise by HQ." was added to the flavor text.

ExcessiveUseOfCobblestone committed:
Update lavaland_ruin_code.dm

KorPhaeron commented:
lmao

bgobandit commented:
it's a catch-22, is the thing, players might swing the other way and use this as an excuse to murderbone because "gotta hinder nanotrasen"

jughu commented:
What if nanotrasen hinders itself?

KorPhaeron commented:
It already says not to leave the base unless its compromised

bgobandit commented:
yeah but given certain "functionalities" syndie base has (like that bug about releasing the singulo) there's still a lot of griff that can happen

KorPhaeron commented:
How can the syndie base release the singulo?

Also they are meant to "griff" in certain ways

bgobandit commented:
https://github.com/tgstation/tgstation/issues/23964

RandomMarine commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -144,5 +144,5 @@
 	mask = /obj/item/clothing/mask/chameleon
 	suit = /obj/item/clothing/suit/armor/vest
-	flavour_text = "<font size=3>You are a syndicate agent, employed in a top secret research facility developing biological weapons. Unfortunatley, your hated enemy, Nanotrasen, has begun mining in this sector. <b>Monitor enemy activity as best you can, and try to keep a low profile. Do not abandon the base without good cause.</b> Use the communication equipment to provide support to any field agen ...
+	flavour_text = "<font size=3>You are a syndicate agent, employed in a top secret research facility developing biological weapons. Unfortunatley, your hated enemy, Nanotrasen, has begun mining in this sector. <b>Monitor enemy activity as best you can, and try to keep a low profile. Do not abandon the base without good cause.</b> Use the communication equipment to provide support to any field agen ...
```
I just noticed there's an extra closing bold tag at the end.
It might be a good idea to add some line breaks and bold the part you added.

ExcessiveUseOfCobblestone committed:
Commit 2

Bolds important bits, Underline is basically the most important [you can read underlined and play the role just fine]

Adds newlines.

ExcessiveUseOfCobblestone commented:
well heck

ExcessiveUseOfCobblestone committed:
+\

Hatfish commented:
Can't release the singulo when there's no singulo to release.

octareenroon91 commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -144,5 +144,9 @@
 	suit = /obj/item/clothing/suit/armor/vest
-	flavour_text = "<font size=3>You are a syndicate agent, employed in a top secret research facility developing biological weapons. Unfortunatley, your hated enemy, Nanotrasen, has begun mining in this sector. <b>Monitor enemy activity as best you can, and try to keep a low profile. Do not abandon the base without good cause.</b> Use the communication equipment to provide support to any field agen ...
+	flavour_text = "<font size=3>You are a syndicate agent, employed in a top secret research facility developing biological weapons. \n +\
+					Unfortunatley, your hated enemy, Nanotrasen, has begun mining in this sector. <b>Monitor enemy activity as best you can, and try to keep a low profile. <u> Do not abandon the base, activate the self destruct device if you are compromised.</u></b> \n +\
```
Spelling fix: 'Unfortunately'

ExcessiveUseOfCobblestone replied:
wew

ExcessiveUseOfCobblestone committed:
Unfortunately, I cannot spell unfortunately.

ExcessiveUseOfCobblestone commented:
I will do oranges the favor of saying Headmins probably need to input on this since it's not so much of a code issue as it is an administrative one.

PKPenguin321 commented:
To be honest I wish that this part of the lavabase was removed so it was less like an undetectable, unkillable antagonist-aligned AI, but I suppose this is an improvement over the old text so 👍

uraniummeltdown commented:
We should just remove the Syndicate Lavabase, if you're not allowed to step outside what's the point? Setting up a personal singleplayer server is easier and you can test out deathmixes a lot faster, it's a waste of a ruin to be honest. Everyone prefers Ash Walkers, Golems and Podpeople to this crap

ExcessiveUseOfCobblestone commented:
That's not true...
